src/lib/utils/Identity.ts

```diff
@@ -0,0 +1,111 @@
+import dateFormat from "dateformat";
+import fs from "fs";
+import path from "path";
+import untildify from "untildify";
+import { IFingerprint, IFingerprintRequest } from "../../types";
+import { Base64 } from "./Base64";
+import { sha256 } from "./crypto";
+import { Fingerprint } from "./Fingerprint";
+import { randomIntFromInterval, randomString, randomString2 } from "./strings";
+
+/**
+ * Used to load Fingerprint
+ *
+ * @public
+ */
+export class Identity {
+    private _filepath: string;
+    private _fingerprint: Fingerprint;
+
+    constructor(filePath: string) {
+        this._filepath = path.resolve(untildify(path.normalize(filePath)));
+
+        // load fingerprint from file
+        const txt = fs.readFileSync(this._filepath).toString();
+        const fpdata = JSON.parse(txt);
```

Comment:
The loaded file should be validated with a schema, I would recommend using [`zod`](https://www.npmjs.com/package/zod) to do the job.